agent-deploy/Dockerfile

```diff
@@ -104,6 +105,31 @@ COPY .curlrc .wgetrc /root/
 RUN git config --global user.email "package@datadoghq.com" && \
     git config --global user.name "Bits"
 
+# Install Mono
+RUN gpg --no-default-keyring --keyring /usr/share/keyrings/mono-official-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF \
+    && echo "deb [signed-by=/usr/share/keyrings/mono-official-archive-keyring.gpg] https://download.mono-project.com/repo/ubuntu stable-bionic main" | tee /etc/apt/sources.list.d/mono-official-stable.list \
+    && chmod 644 /usr/share/keyrings/mono-official-archive-keyring.gpg \
+    && apt update \
+    && apt install -y mono-devel
+
+# Install .NET
+RUN mkdir -p /root/.dotnet \
+    && cd /root/.dotnet \
+    && wget https://download.visualstudio.microsoft.com/download/pr/17b6759f-1af0-41bc-ab12-209ba0377779/e8d02195dbf1434b940e0f05ae086453/dotnet-sdk-6.0.100-linux-x64.tar.gz \
+    && tar -xf dotnet-sdk-6.0.100-linux-x64.tar.gz \
+    && rm -rf dotnet-sdk-6.0.100-linux-x64.tar.gz 
+
+ENV DOTNET_ROOT=/root/.dotnet
+ENV PATH=$PATH:$DOTNET_ROOT:$DOTNET_ROOT/tools
+
+# Install Chocolatey
+RUN git clone https://github.com/chocolatey/choco.git \
+    && cd choco \
+    && ./build.sh \
+    && cp -r ./code_drop/temp/_PublishedApps/choco /opt/chocolatey \
+    && export ChocolateyInstall=/opt/chocolatey \
```

Comment:
This will have no effect from outside of this command, if chocolatey requires it, you'll need to change the `export` to `ENV ChocolateyInstall /opt/chocolatey`